catalyst/dl/utils/swa.py

```diff
@@ -0,0 +1,75 @@
+from typing import List
+from collections import OrderedDict
+import glob
+import os
+from pathlib import Path
+
+import torch
+
+
+def average_weights(state_dicts: List[dict]) -> OrderedDict:
+    """
+    Averaging of input weights.
+
+    Args:
+        state_dicts (List[dict]): Weights to average
+
+    Raises:
+        KeyError: If states do not match
+
+    Returns:
+        Averaged weights
+    """
+    # source https://gist.github.com/qubvel/70c3d5e4cddcde731408f478e12ef87b
+    params_keys = None
+    for i, state_dict in enumerate(state_dicts):
+        model_params_keys = list(state_dict.keys())
+        if params_keys is None:
+            params_keys = model_params_keys
+        elif params_keys != model_params_keys:
+            raise KeyError(
+                "For checkpoint {}, expected list of params: {}, "
+                "but found: {}".format(i, params_keys, model_params_keys)
+            )
+
+    average_dict = OrderedDict()
+    for k in state_dicts[0].keys():
+        average_dict[k] = torch.div(
+            sum(state_dict[k] for state_dict in state_dicts), len(state_dicts),
+        )
+    return average_dict
+
+
+def load_weight(path: str) -> dict:
+    """
+    Load weights of a model.
+
+    Args:
+        path (str): Path to model weights
```

Comment:
could you please remove type from docs (the type already specified in arguments)